openreview/conference/templates/areachairWebfield.js

```diff
@@ -651,12 +672,94 @@ var registerEventHandlers = function() {
   });
 
   $('#group-container').on('click', 'a.collapse-btn', function(e) {
-    $(this).next().slideToggle();
+    // $(this).next().slideToggle();
```

Comment:
is this commented code ok?

Reply:
We do not need to do a toggle here as the widget does it by itself.